### ISSUE TITLE
Active?

### ISSUE THREAD

**Issue by atdiff** [chrisbateman/webpack-visualizer#61]
Is this project still active and being maintained? Or is there another recommended alternative?

**Comment by herodrigues** [comment]
You can  use [webpack-contrib/webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer)

**Comment by chrisbateman** [comment]
No, not really. I'm not sure it's worth maintaining since it's impossible to get accurate stats with the stats.json alone.

If anyone feels like there's a benefit here over webpack-bundle-analyzer though, I'd be interested to hear about it.

**Comment by mikestopcontinues** [comment]
What about using the output of bundle-analyzer, since that produces accurate measurements? Or what about making a PR to add this viz to bundle-analyzer? I just discovered it, and honestly... so much nicer!
